AODN/AODN-WAVE-DM/DES-QLD-WAVE-DM/lib/qld_netcdf.py

```diff
@@ -4,8 +4,8 @@
 import pandas as pd
 from netCDF4 import Dataset, date2num, stringtochar
 
-from generate_netcdf_att import generate_netcdf_att
-from util import get_git_revision_script_url
+from python.generate_netcdf_att import generate_netcdf_att
+from python.util import get_git_revision_script_url
```

Comment:
as above, that should be reverted

Reply:
not know. That's how it is working here for me, I will clean up later